tests/models/instructblip/test_modeling_instructblip.py

```diff
@@ -945,7 +945,7 @@ def test_expansion_in_processing(self):
         # Add args to the config to trigger new logic when inputs are expanded in processing file
         processor.num_query_tokens = model.config.num_query_tokens
         processor.tokenizer.add_special_tokens({"additional_special_tokens": ["<image>"]})
-        model.config.image_token_index = len(processor.tokenizer) - 1
+        model.config.image_token_index = len(processor.tokenizer) - 2
```

Comment:
because the new configs hold both image and video models which are loaded as two different Model Classes in transformers. So the we have two special tokens now and the image token is pre-last, while video token is last id